client/src/index.js

```diff
@@ -0,0 +1,32 @@
+//This is the entry file for react and webpack
+
+import React, {Component} from 'react';
+import ReactDOM from 'react-dom';
+import AppAPI from './js/utils/appAPI'
+import App from "./js/components/App";
+import { BrowserRouter} from
+'react-router-dom';
+
+import {firebaseAuth, firebase}from '../../server/config'
+
```

Comment:
Use this `{ BrowserRouter }` format not `{ BrowserRouter}` or `{BrowserRouter}` in your imports/object deconstruction.